packages/frontend/src/components/DashboardTabs/index.tsx

```diff
@@ -232,122 +232,126 @@ const DashboardTabs: FC<DashboardTabsProps> = ({
                                 }
                             }}
                             style={{
-                                paddingTop: 5,
+                                marginTop: 5,
                             }}
+                            styles={
+                                tabsEnabled
+                                    ? {
+                                          root: {
+                                              backgroundColor: 'white',
+                                          },
+                                      }
+                                    : undefined
+                            }
+                            variant="outline"
                         >
                             {sortedTabs && sortedTabs?.length > 0 && (
-                                <Group
-                                    w="100%"
-                                    noWrap
-                                    position="apart"
-                                    spacing="xs"
-                                    style={
-                                        (sortedTabs && sortedTabs.length > 0) ||
-                                        isEditMode
-                                            ? {
-                                                  background: 'white',
-                                                  padding: 5,
-                                                  borderRadius: 3,
-                                              }
-                                            : undefined
-                                    }
+                                <Tabs.List
+                                    style={{
+                                        backgroundColor:
+                                            'var(--mantine-color-gray-0)',
+                                        paddingLeft: '1.25em',
```

Comment:
Where possible use Mantine [props](https://v6.mantine.dev/styles/style-props/#supported-props) and abstract values (xs, sm, md, lg, xl) that can easily be changed in the theme.

Reply:
Thanks, will look into this. Still getting familiar with Mantine.